src/ellswift.rs

```diff
@@ -291,13 +293,17 @@ impl ElligatorSwiftSharedSecret {
 /// This distinction is important because the different parties compute different
 /// hashes of the shared secret.
 #[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[deprecated(since = "0.30.0", note = "Use `Party` instead")]
+#[allow(deprecated)]
 pub enum ElligatorSwiftParty {
     /// We are the initiator of the ECDH
     A,
     /// We are the responder of the ECDH
     B,
 }
 
+#[allow(deprecated)]
+#[allow(dead_code)] // We aren't using this anymore in this library, but users might be using it
```

Comment:
In c83c5a762a0a014acb02e085f17ea12fb6038c6c:

How can users be using this if it's a private method?

Reply:
Not sure what i was thinking - removed this bit

Reply:
This is not resolved. Now this is dead code and is causing a local CI failure for me.

Reply:
In 3b7d3938873eb62406dd4d52094deeabddb0074f